components/layout/construct.rs

```diff
@@ -250,13 +250,17 @@ impl InlineFragmentsAccumulator {
 
     fn from_inline_node<N>(node: &N, style_context: &SharedStyleContext) -> InlineFragmentsAccumulator
             where N: ThreadSafeLayoutNode {
+        let style = node.style(style_context);
+        let text_style = style_context.stylist.style_for_anonymous_box(
+            &style_context.guards, &PseudoElement::ServoText, &style);
         InlineFragmentsAccumulator {
             fragments: IntermediateInlineFragments::new(),
             enclosing_node: Some(InlineFragmentNodeInfo {
                 address: node.opaque(),
                 pseudo: node.get_pseudo_element_type().strip(),
-                style: node.style(style_context),
+                style: style,
                 selected_style: node.selected_style(),
+                text_style: text_style,
```

Comment:
This is just for caching, right? There's nothing preventing us from computing it on demand. May you document the field?